Formula/p/python@3.9.rb

```diff
@@ -24,6 +24,9 @@ class PythonAT39 < Formula
   # build packages later. Xcode-only systems need different flags.
   pour_bottle? only_if: :clt_installed
 
+  # https://devguide.python.org/versions/#versions
+  deprecate! date: "2025-10-15", because: :deprecated_upstream
```

Comment:
It's actually EOL, not just deprecated then. Only 3.13 isn't deprecated.
```suggestion
  disable! date: "2025-10-15", because: :deprecated_upstream
```

Reply:
Honestly I am not a big fan of disabling packages. I believe there should be a period when a package is deprecated and after some time (f.e. 3 months) it can be disabled